devops/azure-pipelines/discord-job-template.yml

```diff
@@ -1,25 +1,30 @@
+parameters:
+  - name: 'dependOn'
+    type: string
+  - name: 'name'
+    type: string
+  - name: 'title'
+    type: string
+  - name: 'discordVarGroup'
+    type: string
```

Comment:
be consistent about yaml sequence indentation (throughout file)

Reply:
still a problem. this file indents extra but the pipeline file does not for the `trigger` and `pr` mappings.